### PR TITLE
Tests: better mysql connection configuration

### PR DESCRIPTION
**Pull request by foxycode**
Every time I need to rebase from master, that one edited config won't allow it. This way, each developer can create own local config and travis will stil work. Hope you merge it.

### CODE REVIEW

File: tests/KdybyTests/Doctrine/Connection.phpt [hunk 1]
```diff
@@ -34,7 +34,10 @@ class ConnectionTest extends Tester\TestCase
 	protected function loadMysqlConfig()
 	{
 		$configLoader = new Nette\DI\Config\Loader();
-		$config = $configLoader->load(__DIR__ . '/../../mysql.neon', isset($_ENV['TRAVIS']) ? 'travis' : 'localhost');
+		$config = $configLoader->load(__DIR__ . '/../../mysql.neon');
+		if (!isset($_ENV['TRAVIS'])) {
```
Comment (fprochazka) [inline tests/KdybyTests/Doctrine/Connection.phpt]: I like it, but I would make it dependent on the existence of the file, not `_ENV` variable

**Comment by foxycode** [comment]
@fprochazka Updated.

**Comment by foxycode** [comment]
@fprochazka Updated as you wanted and tests passing, will you please merge it?

**Comment by fprochazka** [comment]
@foxycode one comment is enought, I haven't had a change to look at it up untill now :)

**Comment by fprochazka** [comment]
Thank you!

**Comment by foxycode** [comment]
@fprochazka Sorry, I have limited time today and I'd like to do some more work while I can.

THanks for merging.

**Pull request merged**